src/nft/utils/calcPoolPrice.ts

```diff
@@ -43,29 +31,23 @@ export const calcPoolPrice = (asset: GenieAsset, position = 0) => {
 
   const ethReserves = BigNumber.from(
     (
-      (nft?.ethReserves as number) ??
-      (
-        nft as Record<
-          string,
-          {
-            ethReserves: number
-          }
-        >
-      )?.poolMetadata?.ethReserves
-    )?.toLocaleString('fullwide', { useGrouping: false }) ?? 1
+      nft as Record<
+        string,
+        {
+          ethReserves: number
+        }
+      >
+    )?.poolMetadata?.ethReserves?.toLocaleString('fullwide', { useGrouping: false }) ?? 1
   )
   const tokenReserves = BigNumber.from(
     (
-      (nft?.tokenReserves as number) ??
-      (
-        nft as Record<
-          string,
-          {
-            tokenReserves: number
-          }
-        >
-      )?.poolMetadata?.tokenReserves
-    )?.toLocaleString('fullwide', { useGrouping: false }) ?? 1
+      nft as Record<
+        string,
+        {
+          tokenReserves: number
+        }
+      >
+    )?.poolMetadata?.tokenReserves?.toLocaleString('fullwide', { useGrouping: false }) ?? 1
```

Comment:
don't fully understanding this change

Reply:
The sell order data returned from the v3 endpoint was structured differently than from gql. Previously this logic handled both but not only has to handle the one case. We still need to have this typecasting as the protocolParameters are passed back as JSON which we parse on the FE. Talked to BE about having it be more structured but that's in the backlog